### ISSUE TITLE
Option for English translation?

### ISSUE THREAD

**Issue by PlaylistsTrance** [Juna-Idler/RhythmiKaRuTTE#4]
Could you add an English option for the texts? I have been able to figure out how to use the program, because I knew how to use the non-karaoke version and with Google Translate, but it would be helpful still, especially for new users.

**Comment by Juna-Idler** [comment]
切り替え機能を用意するのが面倒なのでとりあえずはこんなものかな？
Since it is troublesome to prepare a switching function, is it something like this for the time being?

**Comment by PlaylistsTrance** [comment]
Yes, this is a good compromise. Thank you for adding some translations.
